lib/maze/logger.rb

```diff
@@ -2,11 +2,15 @@
 
 require 'logger'
 require 'singleton'
+require 'time'
 
 # Logger classes
 module Maze
   # A logger, with level configured according to the environment
   class Logger < Logger
+    # @return [String] A timestamp of when the test run started, using the Logger initialization as a basis
+    attr_reader :start_time
```

Comment:
Although we've added this to support the extraction of logs, it's not related to the `Maze::Logger` class itself.  I think `maze.rb` would be a better place for the new state, just like we have `Maze.driver`.